templates/shared/forms/interactive/kubernetes.html

```diff
@@ -276,19 +77,19 @@ <h3 class="p-heading--two">Thank you for enquiring about Kubernetes</h3>
           </div>
           <div class="col-5 u-vertically-center u-align--center u-hide--small">
             {{
-              image(
-              url="https://assets.ubuntu.com/v1/cbae9a60-thank_you_orange_cmyk.svg",
-              alt="smile",
-              width="200",
-              height="200",
-              hi_def=True,
-              loading="auto",
-              ) | safe
+            image(
```

Comment:
Why remove the indent?

Reply:
It wasn't intentional, but it's fixed now